artemis-server/src/main/java/org/apache/activemq/artemis/core/server/LargeServerMessage.java

```diff
@@ -27,6 +27,8 @@
 
 public interface LargeServerMessage extends ReplicatedLargeMessage {
 
+   boolean DEBUG = Boolean.parseBoolean(System.getProperty("ARTEMIS_LARGE_MESSAGE_DEBUG", "false"));
```

Comment:
I will just use logger.isDebugEnabled();

if RefCountMessage.logger is debug, it will be using debug


Maybe I will also expose the logger from RefCountMessage with RefCountMessage::isDebugEnabled()

As I will be adding a method refCountMessage.addDebug(String );

(Not sure I will keep this method in the final version after I figure out the issue I'm dealing with now...
  apparently a message could be duplicated after a failed AMQP Consumer in some weird race, and if I add logger.debug() the error is gone... the only way I have been able to capture this was by adding a new event to the debugState, and checking if I'm trying to remove the message twice.